openstack/clustering/v1/policies/results.go

```diff
@@ -0,0 +1,89 @@
+package policies
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// Policy represents a clustering policy in the Openstack cloud
+type Policy struct {
+	CreatedAt   *time.Time             `json:"-"`
+	Data        map[string]interface{} `json:"data"`
+	DomainUUID  string                 `json:"domain"`
```

Comment:
Let's change this to just `Domain` to match the JSON field. Same with the other UUID fields here.

Reply:
Np.  I'll fix that.